chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -78,7 +79,7 @@ pub struct DistributeStateWitnessRequest {
     pub epoch_id: EpochId,
     pub chunk_header: ShardChunkHeader,
```

Comment:
do you know why chunk header and epoch id are needed here? those are already included as part of the witness, right?

Reply:
not sure, I will check later (at least clippy is not complaining)

Reply:
looks like the intention was to remove them from the witness once the partial witness distribution was implemented. see the TODOs here: https://github.com/near/nearcore/blob/587c357cb68129663ac2b8a9a5f69fb14c51809c/core/primitives/src/stateless_validation/state_witness.rs#L99

Reply:
I think chunk header is essential there

Reply:
Originally I had the todo there as the chunk header can be determined by the (shard_id, epoch_id, height) triplet which is part of partial witness message

Reply:
sent out https://github.com/near/nearcore/pull/12393 for this.